module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -378,4 +378,27 @@ public function findInThemes($relativePathPattern)
         }
         return array_values($results);
     }
+
+    /**
+     * Search vendor directory for a file.
+     *
+     * @param string $filePath Relative path to file
```

Comment:
If this is a relative path, then can we safely rely on file_exists / is_file below? Do we need to do something to make it absolute first? Or is the comment wrong?